lib/newman_conway.py

```
@@ -1,10 +1,24 @@
-
-
 # Time complexity: ?
 # Space Complexity: ?
 def newman_conway(num):
-    """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
-    """
-    pass
+    if num == 0:
+        raise ValueError
```

We should raise this error for _any_ value below the valid starting point of the sequence:
```py
    if num <= 0:
        raise ValueError
```